gamedata/deploy.rinkeby.json

```diff
@@ -1,25 +1,25 @@
 {
-  "0": "0xba6F0B5784B6580790584A553f6e4a3483a915c3",
-  "1": "0x42E7014a9D1f6765e76fA2e69532d808F2fe27E3",
-  "2": "0x50E1785EeE794253c5E33B8fE123e77124736e38",
-  "3": "0xBB49f532586618586df966c8407C65fEDb2Fe313",
-  "4": "0x990f8E9fb0a9f990F064D863A35FDE6E68808Aff",
-  "5": "0xBd886a37faD1f596221f33ca568122815ED48c81",
-  "6": "0xe04f955e4Cf9858F8f8d60C09aBf16DF23D4672b",
-  "7": "0x492e18ddBd7591638453d2f1B1847F86711105C8",
-  "8": "0xc06a07059564DF5034AfA6d3C8BB144663cbFE2e",
-  "9": "0x8E500A9082D26dfA7CCdecf0391E0b93B9470266",
-  "10": "0x4209f564b6fDB63B34866CEa4B43BF333BcAAAD9",
-  "11": "0x716747Fbc1FcE4c36F2B369F87aDB5D4580e807f",
-  "12": "0x972A4Eb8fb1FfF6e9EF62Cfe5FF4A00519365Ee2",
-  "13": "0xB0815f55B71ede74ACcF7ca17de5E0931ba5BA12",
-  "14": "0x00200f9AeBA83B4bddddd7620569C15AC09663cc",
-  "15": "0x19416E8a80660FE4A6f88a12d28BEb729A349052",
-  "16": "0x71ADA81382C62BB1D5CC194c2BD836Eee34AffF5",
-  "18": "0xee776691908C4B3dCc990B98572e685060E3bc9C",
-  "19": "0xDA1f7C628abd817a91c0124245504365E8D93Ee3",
-  "20": "0xae8ed765dbd45Ce48ebBd2496CeD6B1Ee29466fc",
-  "21": "0x865e167C3db2c3E1C046cC67b05E6EcF9C897C2D",
-  "22": "0x32FFB8d4244B350F5D3E074e9b731A135531B975",
-  "ethernaut": "0xD991431D8b033ddCb84dAD257f4821E9d5b38C33"
+  "0": "0x4E73b858fD5D7A5fc1c3455061dE52a53F35d966",
+  "1": "0x9CB391dbcD447E645D6Cb55dE6ca23164130D008",
+  "2": "0x5732B2F88cbd19B6f01E3a96e9f0D90B917281E5",
+  "3": "0x4dF32584890A0026e56f7535d0f2C6486753624f",
+  "4": "0x0b6F6CE4BCfB70525A31454292017F640C10c768",
+  "5": "0x63bE8347A617476CA461649897238A31835a32CE",
+  "6": "0x9451961b7Aea1Df57bc20CC68D72f662241b5493",
+  "7": "0x22699e6AdD7159C3C385bf4d7e1C647ddB3a99ea",
+  "8": "0xf94b476063B6379A3c8b6C836efB8B3e10eDe188",
+  "9": "0x5cECE66f3EB19f7Df3192Ae37C27D96D8396433D",
+  "10": "0x848fb2124071146990c7abE8511f851C7f527aF4",
+  "11": "0xaB4F3F2644060b2D960b0d88F0a42d1D27484687",
+  "12": "0x11343d543778213221516D004ED82C45C3c8788B",
+  "13": "0x9b261b23cE149422DE75907C6ac0C30cEc4e652A",
+  "14": "0xdCeA38B2ce1768E1F409B6C65344E81F16bEc38d",
+  "15": "0x096bb5e93a204BfD701502EB6EF266a950217218",
+  "16": "0x97E982a15FbB1C28F6B8ee971BEc15C78b3d263F",
+  "18": "0x8d07AC34D8f73e2892496c15223297e5B22B3ABE",
+  "19": "0x200d3d9Ac7bFd556057224e7aEB4161fED5608D0",
+  "20": "0xda5b3Fb76C78b6EdEE6BE8F11a1c31EcfB02b272",
+  "21": "0xcE1BB92eeb71AF5Fec09D38B0c854d55285f6e04",
+  "22": "0xBE732789f2963E0522719F2D3fB55E6bfe07e92e",
+  "ethernaut": "0x6B3e148f9745c9847881469dEC2eaD8Fe85175cE"
```

Comment:
If we deploy a new `Ethernaut` contract we would lose the registered levels for people who are mid game. Can we keep the previous one?

Reply:
Yeah it would wipe all the 0.5 levels. We could make either an 05 or 06 subdomain and then host the other on master?

Reply:
Oh I just saw your other comment above. It definitely loses track of which levels you've completed because the factory will be gone

Reply:
Have reverted the address to the old one and registered the new levels